model/task/task.go

```diff
@@ -2073,17 +2073,42 @@ func (t *Task) MergeNewTestResults() error {
 	return nil
 }
 
-// GetTestResultsForDisplayTask returns the test results for the execution tasks
-// for a display task.
+// MergeTestResults returns the task with both old (embedded in the tasks
+// collection) and new (from the testresults collection) OR Cedar test results
+// merged in the Task's LocalTestResults field.
+func (t *Task) MergeTestResults() error {
```

Comment:
I think we need to switch to this new function [here too](https://github.com/evergreen-ci/evergreen/blob/777962000715c87f2844d9c7275261436d90e1d2/trigger/task.go#L807)